sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/ConfigurationSamples.cs

```diff
@@ -0,0 +1,189 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Iot.Hub.Service.Models;
+
+namespace Azure.Iot.Hub.Service.Samples
+{
+    /// <summary>
+    /// This sample goes through the usage of Configuration on the IoT Hub.
+    /// </summary>
+    internal class ConfigurationSamples
+    {
+        public readonly IotHubServiceClient IoTHubServiceClient;
+        public const int MaxRandomValue = 200;
+        public static readonly Random Random = new Random();
+
+        public ConfigurationSamples(IotHubServiceClient client)
+        {
+            IoTHubServiceClient = client;
+        }
+
+        public async Task RunSampleAsync()
+        {
+            string testConfigurationId = $"configuration{Random.Next(MaxRandomValue)}";
+            TwinConfiguration twinConfiguration = CreateSampleConfig(testConfigurationId);
+            
+            // Create a Twin Configuration.
+            await CreateConfiguratioAsync(twinConfiguration);
```

Comment:
currently adding sample for CRUD, we will modify and add edge config and test queries when we have implementation for the same.